exporters/prometheus/test/exporter_utils_test.cc

```diff
@@ -81,18 +81,21 @@ void assert_histogram(prometheus_client::MetricFamily &metric,
 
 TEST(PrometheusExporterUtils, TranslateToPrometheusEmptyInputReturnsEmptyCollection)
 {
-  auto translated = PrometheusExporterUtils::TranslateToPrometheus(
-      std::vector<std::unique_ptr<metric_sdk::ResourceMetrics>>{});
+  metric_sdk::ResourceMetrics metrics_data = {};
+  auto translated = PrometheusExporterUtils::TranslateToPrometheus(metrics_data);
   ASSERT_EQ(translated.size(), 0);
 }
 
+#if 0
```

Comment:
Would it possible to change the unit tests and not disable them? Or just create a new issue for the unit test?

Reply:
Thanks, have changed the unit tests now.